src/index.html

```diff
@@ -1,14 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo-header">
+      <a href="#" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="logo-img"
+        >
+      </a>
+      <nav class="moyo-nav">
+        <a href="#" class="nav-links is-active">apple</a>
```

Comment:
```suggestion
        <a href="#" class="nav-links is-active">Apple</a>
```
it's better to upper case first